t/Test27-multiple_nests.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bats
+
+#***********************************************************************
+#                   GNU Lesser General Public License
+#
+# This file is part of the GFDL FRE NetCDF tools package (FRE-NCTools).
+#
+# FRE-NCTools is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or (at
+# your option) any later version.
+#
+# FRE-NCTools is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with FRE-NCTools.  If not, see
+# <http://www.gnu.org/licenses/>.
+#***********************************************************************
+
+# Test grid for multiple same level and telescoping nests 
+
+@test "Test grid for multiple same level and telescoping nests" {
+
+  if [ ! -d "Test27" ] 
+  then
+  		mkdir Test27
+  fi
+
+  cd Test27
+
+#Make_hgrid: create three same level -level1- nests in tiles 2,5,6"
+  run command make_hgrid \
+		--grid_type gnomonic_ed \
+		--nlon 96 \
+		--grid_name C48_grid \
+		--do_schmidt \
+		--stretch_factor 1.0 \
+		--target_lon -97.5 \
+		--target_lat 36.5 \
+		--nest_grids 3 \
```

Comment:
Hi @JosephMouallem `--nest_grids` should be singular, `--nest_grid` here and below in line 66